src/renderer.cpp

```diff
@@ -1216,14 +1478,9 @@ uint32_t Renderer::blitTexture(uint32_t x, uint32_t y, uint32_t width, uint32_t
         }
     }
 
-    backendViewId++;
-
-    bgfx::blit(backendViewId, ret, 0, dstY, bgfx::getTexture(backendFrameBuffer), newX, newY, newWidth, newHeight);
-    bgfx::touch(backendViewId);
-    setClearFlags(false, false);
-
-    backendViewId++;
-    setClearFlags(false, false);
+    bgfx::blit(VIEW_ID_BLIT, ret, 0, dstY, bgfx::getTexture(backendFrameBuffer), newX, newY, newWidth, newHeight);

```

Comment:
Same here, as said previous, the idea of a progressive ID is exactly to sort blits where they are supposed to be. In FF7 you'll not notice it, but on FF8 this code would kill a lot of scenes ( you will notice it the most during the final battle )

Reply:
Please see comment above. Unfortunately I do not own a copy of FF8 right now so I was not able to check if something is broken.

Reply:
As explained in the previous comment, I reverted to using the `backendViewId` variable.